alarm/kodi-rbp4/kodi.install

```diff
@@ -0,0 +1,26 @@
+post_install() {
+  [[ $(type -p gtk-update-icon-cache) ]] && usr/bin/gtk-update-icon-cache -qtf usr/share/icons/hicolor
+  [[ $(type -p update-desktop-database) ]] && usr/bin/update-desktop-database -q usr/share/applications
+  getent group kodi > /dev/null || groupadd -r kodi
+  getent passwd kodi > /dev/null || useradd -r -m -d /var/lib/kodi -g kodi -s /usr/bin/nologin kodi
+  usermod -a -G kodi,audio,video,power,network,optical,storage,disk,tty,input kodi
+  mkdir -p var/lib/kodi
+  chown -R kodi:kodi var/lib/kodi
+
+  echo "****************************************************************"
+  echo "Make sure the following lines are added to /boot/config.txt"
```

Comment:
Per our email regarding filtering for this file, we could omit this and just include a separate file this package would provide as a drop-in but that would require modifying `/boot/config.txt` provided by linux-raspberrypi

What do you think?